opensrp-chw/build.gradle

```diff
@@ -249,14 +250,15 @@ dependencies {
 //        exclude group: 'id.zelory', module: 'compressor'
 //    }
 
-    implementation('org.smartregister:opensrp-client-chw-referral:1.3.13-NACP-ALPHA-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-chw-referral:1.3.13-NACP-LOCAL-ALPHA-SNAPSHOT@aar') {
```

Comment:
@nKataraia - did you make any changes to the referral library? If not probably just leave this to the version others are using locally.